libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm/differentNeedsSection/healthProtectionSubSection.ts

```diff
@@ -212,6 +264,9 @@ export const healthProtectionSubSection = buildSubSection({
             )
           },
         }),
+        buildHiddenInput({
+          id: 'healthProtection.triggerHiddenInput',
+        }),
```

Comment:
Maybe add `doesNotRequireAnswer`?
```suggestion
        buildHiddenInput({
          id: 'healthProtection.triggerHiddenInput',
          doesNotRequireAnswer: true,
        }),
```

Reply:
Fixed